### PR TITLE
fix: add Arabic Jordanian General Tweets (AJGT) corpus

### PR DESCRIPTION
**Pull request by Akash190104**
Added the Arabic Jordanian General Tweets (AJGT) Corpus
## Checklist for adding MMTEB dataset

Reason for dataset addition:

Consists of Jordanian dialect in the arabic language.

- [x] I have tested that the dataset runs with the `mteb` package.
- [x] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb run -m {model_name} -t {task_name}` command.
  - [x] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [x] `intfloat/multilingual-e5-small`
- [x] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [x] If the dataset is too big (e.g. >2048 examples), considering using `self.stratified_subsampling() under dataset_transform()`
- [x] I have filled out the metadata object in the dataset file (find documentation on it [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md#2-creating-the-metadata-object)).
- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 
- [x] I have added points for my submission to the [points folder](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md) using the PR number as the filename (e.g. `438.jsonl`).

### CODE REVIEW

File: mteb/tasks/Classification/ara/AJGT.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  title={Arabic tweets sentimental analysis using machine learning},
+  author={Alomari, Khaled Mohammad and ElSherif, Hatem M and Shaalan, Khaled},
+  booktitle={International Conference on Industrial, Engineering and Other Applications of Applied Intelligent Systems},
+  pages={602--610},
+  year={2017},
+  organization={Springer}
+}
+""",
+        n_samples={"train": 1800},
+        avg_character_length={"test": 46.81},
```
Comment (asparius) [inline mteb/tasks/Classification/ara/AJGT.py:41]: should be train

**Review by asparius — CHANGES_REQUESTED** [review]
Change the key in avg_character_length metadata, otherwise looks fine.

**Comment by Akash190104** [comment]
@asparius added points as well, and fixed the key issue. Please merge this. Thanks

**Comment by asparius** [comment]
thanks for your contributions @Akash190104, I dont have a write access to merge pr into repo though. @KennethEnevoldsen @isaac-chung

**Pull request merged**